src/diffusers/models/autoencoder_kl.py

```diff
@@ -219,7 +223,12 @@ def tiled_encode(self, x: torch.FloatTensor, return_dict: bool = True) -> Autoen
         for i in range(0, x.shape[2], overlap_size):
             row = []
             for j in range(0, x.shape[3], overlap_size):
-                tile = x[:, :, i : i + self.tile_sample_min_size, j : j + self.tile_sample_min_size]
+                tile = x[
```

Comment:
Let's not reformat this please :-)

Reply:
Hmm, not sure why, but black 23.1 keeps formatting this line. I'll leave it out of the commit though